### ISSUE TITLE
bug(RVX Manager): patching fails when updating RVX in root environment

### ISSUE THREAD

**Issue by Rairof** [inotia00/ReVanced_Extended#2278]
### Type

Error while patching

### Tools used

RVX Manager

### Application

YouTube 19.16.39

### Bug description

App patching fails when app is selected from "Select apps" option directly instead of selecting.apk file from storage.

### Error logs

```shell
- Device Info
ReVanced Manager: 1.20.4
Model: 2107113SG
Android version: 14
Supported architectures: arm64-v8a, armeabi-v7a, armeabi
Root permissions: Yes

- Patch Info
App: com.google.android.youtube v19.16.39 (Suggested: 19.16.39)
Patches version: v4.10.3
Patches added: Change version code [Change version code: true], Custom header for YouTube, Force hide player buttons background, Hide Shorts dimming, MaterialYou, Visual preferences icons [RVX settings menu icon: gear]
Patches removed: Theme, GmsCore support
Default patch options changed: Custom Shorts action buttons [Shorts icon style : round], Custom branding icon for YouTube [App icon: youtube], Custom branding name for YouTube [App name: YouTube], Translations for YouTube [Translations to add: ar, el-rGR, es-rES, fr-rFR, hu-rHU, it-rIT, ja-rJP, ko-rKR, pl-rPL, pt-rBR, ru-rRU, tr-rTR, uk-rUA, vi-rVN, zh-rCN, zh-rTW], Overlay buttons [Icon type: rounded], Settings for YouTube [RVX settings menu name: Addons], SponsorBlock [Outline icons: true]

- Settings
Allow changing patch selection: true
Version compatibility check: false
Show universal patches: true
Remove unused library: true
Use pre-release: true
Patches source: inotia00/revanced-patches
Integration source: inotia00/revanced-integrations

- Logs
Reading APK
Decoding app manifest
Loading patches
Merging integrations
Deleting existing temporary files directory
Decoding resources
Executing patches
Alternative thumbnails failed: app.revanced.patcher.patch.PatchException: 'Alternative thumbnails' depends on 'NavigationBarHookPatch' that raised an exception:
app.revanced.patcher.patch.PatchException: Failed to resolve PivotBarButtonsViewSetSelectedFingerprint
	at app.revanced.util.BytecodeUtilsKt.getException(BytecodeUtils.kt:43)
	at app.revanced.util.BytecodeUtilsKt.resultOrThrow(BytecodeUtils.kt:35)
	at app.revanced.patches.youtube.utils.navigation.NavigationBarHookPatch.execute(NavigationBarHookPatch.kt:102)
	at app.revanced.patches.youtube.utils.navigation.NavigationBarHookPatch.execute(NavigationBarHookPatch.kt:30)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:190)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:91)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)

	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:148)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)
Applied 25 patches
Description components failed: app.revanced.patcher.patch.PatchException: 'Description components' depends on 'LithoFilterPatch' that raised an exception:
app.revanced.patcher.patch.PatchException: Failed to resolve SetByteBufferFingerprint
	at app.revanced.util.BytecodeUtilsKt.getException(BytecodeUtils.kt:43)
	at app.revanced.util.BytecodeUtilsKt.resultOrThrow(BytecodeUtils.kt:35)
	at app.revanced.patches.shared.litho.LithoFilterPatch.execute(LithoFilterPatch.kt:57)
	at app.revanced.patches.shared.litho.LithoFilterPatch.execute(LithoFilterPatch.kt:29)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:190)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:91)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)

	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:148)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)
Enable open links directly failed: app.revanced.patcher.patch.PatchException: Target index not found. startIndex: 0, name: parse
	at app.revanced.util.BytecodeUtilsKt.checkIndex(BytecodeUtils.kt:452)
	at app.revanced.util.BytecodeUtilsKt.getTargetIndexWithMethodReferenceNameOrThrow(BytecodeUtils.kt:403)
	at app.revanced.patches.youtube.misc.openlinksdirectly.OpenLinksDirectlyPatch.execute(OpenLinksDirectlyPatch.kt:35)
	at app.revanced.patches.youtube.misc.openlinksdirectly.OpenLinksDirectlyPatch.execute(OpenLinksDirectlyPatch.kt:16)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:190)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)
Fullscreen components failed: app.revanced.patcher.patch.PatchException: 'Fullscreen components' depends on 'LithoFilterPatch' that raised an exception:
app.revanced.patcher.patch.PatchException: 'LithoFilterPatch' did not succeed previously
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:45)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:91)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)

	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:148)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)
Hide action buttons failed: app.revanced.patcher.patch.PatchException: 'Hide action buttons' depends on 'LithoFilterPatch' that raised an exception:
app.revanced.patcher.patch.PatchException: 'LithoFilterPatch' did not succeed previously
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:45)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:91)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)

	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:148)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)
Hide ads failed: app.revanced.patcher.patch.PatchException: 'Hide ads' depends on 'AdsBytecodePatch' that raised an exception:
app.revanced.patcher.patch.PatchException: Failed to resolve CompactYpcOfferModuleViewFingerprint
	at app.revanced.util.BytecodeUtilsKt.getException(BytecodeUtils.kt:43)
	at app.revanced.util.BytecodeUtilsKt.resultOrThrow(BytecodeUtils.kt:35)
	at app.revanced.patches.youtube.ads.general.AdsBytecodePatch.execute(AdsBytecodePatch.kt:58)
	at app.revanced.patches.youtube.ads.general.AdsBytecodePatch.execute(AdsBytecodePatch.kt:26)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:190)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:91)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Unknown Source:73)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source:0)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Unknown Source:6)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source:0)
	at app.revanced.manager.flutter.MainActivity.runPatcher$lambda$35(Unknown Source:459)
	at app.revanced.manager.flutter.MainActivity.$r8$lambda$0x5RQW3TgU-norbj4KLWHG0V2oA(SourceFile:0)
	at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda4.run(SourceFile:0)
	at java.lang.Thread.run(Thread.java:1012)

	at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Unknown Source:148)
	at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Unknown Source:292)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at app.revanced.patcher.Patcher$apply$1.invoke(SourceFile:0)
	at kotlinx.coroutines.flow.SafeFlow.collectSafely(Unknown Source:2)
	at kotlinx.coroutines.flow.AbstractFlow.collect(Unknown Source:71)
	at app.revanced.manager.flutter.MainActivity$runPatcher$1$patcherResult$1$2.invokeSuspend(Unknown Source:76)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:11)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:93)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(Unknown Source:46)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Unknown Source:23)
	at kotlinx.corout
```

### Solution

Donno ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯

### Additional context

Same patch fail happens with YouTube Music.
Here are the patch logs-
https://katb.in/uxawewonefo

### Device Environment

Android 14 QPR3, RisongOS 4.0

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [ ] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by Rairof** [comment]
By "through installed app" I mean through this menu-
![IMG_20240729_005845](https://github.com/user-attachments/assets/58a96bb2-9960-43a2-bc29-74341a2eb197)

**Comment by Rairof** [comment]
This might be unrelated and a different issue but Reddit also fails to patch now-
![Screenshot_20240729-013551_Pixel Launcher](https://github.com/user-attachments/assets/28112cd4-cbc2-4f99-a1f0-2bca15955aee)

**Comment by inotia00** [comment]
Here is the process I was able to reproduce on my side:
1. Install RVX Manager, and perform the patch in root environment for the first time.
2. Launch RVX, and verify that the patch is applied.
3. Re-open RVX Manager, and perform the patch in root environment again.
4. The issue is reproduced.

This is probably an error on the official ReVanced Manager side.

A temporary workaround is to follow the steps mentioned in the [documentation](https://github.com/inotia00/revanced-documentation/blob/main/docs/rvx-manager.md#using-rvx-manager), which should resolve the issue.
(i.e. install an unpatched YouTube before updating RVX).

A more fundamental solution would be to release a new RVX Manager build that includes the fix.

**Comment by Rairof** [comment]
https://github.com/inotia00/ReVanced_Extended/issues/2278#issuecomment-2254572299
That seems to solve the issue but break custom branding patch.

**Comment by inotia00** [comment]
If you use this method, you will need to restart your device after updating RVX (for Custom branding patch)

**Comment by Rairof** [comment]
I will try it now. Thank you.

**Comment by Rairof** [comment]
https://github.com/inotia00/ReVanced_Extended/issues/2278#issuecomment-2254573878
Restarting device doesn't seem to always work.
Turning on themed icons on and off in wallpapers & styles menu seemed to work in my case. Thank you regardless.

**Comment by inotia00** [comment]
fixed in [rvx-manager-1.21.1](https://github.com/inotia00/revanced-manager/releases/tag/v1.21.1)
